### ISSUE TITLE
New Shipment emails do not generate

### ISSUE THREAD

**Issue by DigitalStartupUK** [magento/magento2#22788]
### Preconditions (*)

1. Magento 2.3.1 Open Source
2. Ubuntu 16.04, PHP 7.2, Apache 2.4.18

### Steps to reproduce (*)

1. Go to Marketing > Email Templates
2. Add Template
3. Load magento_sales > New Shipment (Default Template with no custom changes)
4. Press Load
5. Preview Template

### Expected result (*)

1. The Template Preview Tab loads with the default contents

### Actual result (*)

1. The Template Preview Tab is blank. The only HTML source code is below
````
<html lang="en"><head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <title>Email Preview</title>
</head>
<body>
    </body>
</html>
````

**Further Notes:** When you remove the below line from the Default Template, the Template Preview loads:

    {{block class='Magento\\Framework\\View\\Element\\Template' area='frontend' template='Magento_Sales::email/shipment/track.phtml' shipment=$shipment order=$order}}

![template-ok](https://user-images.githubusercontent.com/24552648/57377403-40f1c700-719a-11e9-9c88-11ec37be89c4.png)

**Comment by deepikajaniyani** [comment]
@magento-engcom-team give me 2.3-develop instance 

As checked on a vanilla Magento 2.3.1 instance this issue replicates.

**Comment by magento-engcom-team** [comment]
Hi @deepikajaniyani. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @deepikajaniyani, here is your [Magento instance](https://i-22788-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22788-2-3-develop.instances.magento-community.engineering/admin](https://i-22788-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by shikhamis11** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by shikhamis11** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11, here is your [Magento instance](https://i-22788-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-22788-2-2-develop.instances.magento-community.engineering/admin](https://i-22788-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-99611`, `MAGETWO-99612` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by deepikajaniyani** [comment]
@magento-engcom-team this issue replicates on 2.3.x version, please add the label. I am not able to do so.

**Comment by subbu1993** [comment]
@magento-engcom-team  please give me 2.3.x instance

**Comment by magento-engcom-team** [comment]
Hi @DigitalStartupUK. Thank you for your report. 
The issue has been fixed in magento/magento2#22906 by @maheshWebkul721 in 2.2-develop branch
Related commit(s):
 - [a387566bfd91d0dca7c9955dc808e16175842c6a](https://github.com/magento/magento2/commit/a387566bfd91d0dca7c9955dc808e16175842c6a)

The fix will be available with the upcoming 2.2.10 release.
